This fixes the error: 
	xterm.c:8875: error: structure has no member named `scroll_mode'

A variation of Richard Stallman's patch, see http://lists.gnu.org/archive/html/emacs-devel/2003-11/msg00082.html

--- ./src/xterm.c~	2002-10-15 16:21:45.000000000 +0200
+++ ./src/xterm.c	2006-03-29 02:12:52.000000000 +0200
@@ -118,7 +118,7 @@
 #ifdef HAVE_XAW3D
 #include <X11/Xaw3d/Simple.h>
 #include <X11/Xaw3d/Scrollbar.h>
-#define ARROW_SCROLLBAR
+#define XAW_ARROW_SCROLLBARS
 #include <X11/Xaw3d/ScrollbarP.h>
 #else /* !HAVE_XAW3D */
 #include <X11/Xaw/Simple.h>
